crates/forge/test_utils/src/runner.rs

```diff
@@ -97,12 +97,16 @@ impl Contract {
             .find(|package| package.name == "contract")
             .unwrap();
 
-        let contract =
-            get_contracts_artifacts_and_source_sierra_paths(&scarb_metadata, &package.id, None)
-                .unwrap()
-                .remove(&self.name)
-                .ok_or(anyhow!("there is no contract with name {}", self.name))?
-                .0;
+        let contract = get_contracts_artifacts_and_source_sierra_paths(
+            &scarb_metadata,
+            &package.id,
+            None,
+            false,
```

Comment:
Why is it hardcoded here?

Reply:
This is used for tests. We build the contracts above using `scarb build`, currently at version `2.7.0`. Generally this interface, which uses boolean to decide whether artifacts should be taken from the test target is not ideal. An alternative could be to split `get_contracts_artifacts_and_source_sierra_paths` into multiple functions, but this would require more effort.

Reply:
Okay, I didn't notice this was in test utils